lib/debug/server_cdp.rb

```diff
@@ -716,9 +737,28 @@ def process_protocol_request req
             frame_id = ref[1]
             fid = @frame_map[frame_id]
             request_tc [:cdp, :scope, req, fid]
+          when 'global'
+            vars = global_variables.sort.map do |name|
+              gv = eval(name.to_s)
+              prop = {
```

Comment:
Perhaps we can extract `ThreadClient#propertyDescriptor` and use it here too?

Reply:
Yes, I plan to refactor them after releasing the new version. I added the comment. Thank you.